src/Accelerators/NNPA/Conversion/ZLowToLLVM/ZLowToLLVMCommon.cpp

```diff
@@ -23,6 +23,9 @@
 
 using namespace mlir;
 
+namespace onnx_mlir {
+namespace zlow {
+
```

Comment:
One alternative would be to "use" several using declarations like this: 
```
using namespace mlir;
namespace onnx_mlir;
namespace onnx_mlir::zlow;

void A::foo() { ... }
```

Now is unclear to which namespace the definition of `A::foo()` belongs to.  Wrapping the declaration of `A::foo()` in a namespace makes it clear that it belongs to the onnx_mlir::zlow namespace. 

See also: 
  - https://stackoverflow.com/questions/8681714/correct-way-to-define-c-namespace-methods-in-cpp-file
  - https://stackoverflow.com/questions/10816600/c-namespaces-how-to-use-in-header-and-source-files-correctly

Reply:
Not sure if you mean we should nest  `onnx_mlir` inside `mlir`.   My opinion on that would be that we should not, because we are `using` MLIR but we are not part of MLIR (we aren't a project in tree).

Reply:
I see places with 
```
using namespace mlir;
using namespace std;
```
that does not imply one included in the other ones.. it's just that the namespace is used for everything. I don't see the point of the `namespace {}` when it includes the whole file... if its for a subset, absolutely; if for the whole file, it seems to be a pedantic use of scope.